src/constants/playlists.js

```diff
@@ -1,14 +1,14 @@
 module.exports = [
-  {name: 'favorites', author: 'You!', title: 'Your Saved Favorites'},
```

Comment:
It seems there are still some formatting isssues. See space around `{`

Reply:
there are spaces around the newly added lines, that's consistent with semistandard

Reply:
Objects should be `{name: 'favorites', author: 'You!', title: 'Your Saved Favorites'}` no space before / after curly braces